factory/consensus/consensusComponents.go

```diff
@@ -502,6 +503,7 @@ func (ccf *consensusComponentsFactory) createArgsBaseAccountsSyncer(trieStorageM
 		TrieSyncerVersion:         ccf.config.TrieSync.TrieSyncerVersion,
 		CheckNodesOnDisk:          ccf.config.TrieSync.CheckNodesOnDisk,
 		StorageMarker:             storageMarker.NewTrieStorageMarker(),
+		SyncStatisticsHandler:     statistics.NewTrieSyncStatistics(),
```

Comment:
by providing a new pointer here we assure that no trie sync information will be sent on the heartbeat, if, for example, the node syncs blocks but starts an ad-hoc trie sync process because it finds a missing trie node.